src/main/scala/zio/prelude/CommutativeBoth.scala

```diff
@@ -94,29 +111,44 @@ object CommutativeBoth extends LawfulF.Invariant[CommutativeBothDeriveEqualInvar
           case (Some(a), Some(b)) => Some((a, b))
           case _                  => None
         }
+
+      def bothPar[A, B](fa: => Option[A], fb: => Option[B]): Option[(A, B)] =
+        (fa, fb) match {
+          case (Some(a), Some(b)) => Some((a, b))
+          case _                  => None
+        }
     }
 
   /**
    * The `CommutativeBoth` instance for And `Schedule`.
    */
-  implicit def ScheduleAndCommutativeBoth[R, E]
-    : CommutativeBoth[({ type lambda[+a] = AndF[Schedule[R, E, a]] })#lambda] =
-    new CommutativeBoth[({ type lambda[+a] = AndF[Schedule[R, E, a]] })#lambda] {
+  implicit def ScheduleAndCommutativeBoth[R, E]: CommutativeBoth[Lambda[`+a` => AndF[Schedule[R, E, a]]]] =
```

Comment:
I'm afraid that if the `+a` is in between the back-ticks, it won't be taken as covariant -- are you sure this will work?
It's probably for the best, if we stick to the `lambda` trick, because it works across all Scala versions.